src/Rezoom.SQL.Provider/Rezoom.SQL.Provider.fsproj

```diff
@@ -69,37 +24,17 @@
     <Compile Include="TypeGeneration.fs" />
     <Compile Include="AssemblyResolver.fs" />
     <Compile Include="Provider.fs" />
-    <Content Include="packages.config" />
+    <Content Include="paket.references" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\Rezoom.SQL.Compiler\Rezoom.SQL.Compiler.fsproj">
-      <Name>Rezoom.SQL.Compiler</Name>
-      <Project>{87fcd04a-1f90-4d53-a428-cf5f5c532a22}</Project>
-      <Private>True</Private>
-    </ProjectReference>
+    <ProjectReference Include="..\Rezoom.SQL.Compiler\Rezoom.SQL.Compiler.fsproj" />
+    <ProjectReference Include="..\Rezoom.SQL.Mapping\Rezoom.SQL.Mapping.fsproj" />
     <Reference Include="mscorlib" />
-    <Reference Include="FSharp.Core, Version=$(TargetFSharpCoreVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="Rezoom">
-      <HintPath>..\packages\Rezoom.0.4.2\lib\net45\Rezoom.dll</HintPath>
-    </Reference>
     <Reference Include="System" />
     <Reference Include="System.Configuration" />
     <Reference Include="System.Core" />
     <Reference Include="System.Data" />
     <Reference Include="System.Numerics" />
-    <ProjectReference Include="..\Rezoom.SQL.Mapping\Rezoom.SQL.Mapping.fsproj">
-      <Name>Rezoom.SQL.Mapping</Name>
-      <Project>{6b6a06c5-157a-4fe3-8b4c-2a1ae6a15333}</Project>
-      <Private>True</Private>
```

Comment:
This looks like it shouldn't be omitted.

Reply:
@Swoorup just above: `<ProjectReference Include="..\Rezoom.SQL.Mapping\Rezoom.SQL.Mapping.fsproj" />`  I don't see it omitted, am I missing something?

Reply:
I meant the `<Private>True</Private>` does have some influence on the packaging it seems.